### ISSUE TITLE
When releasing this to AMO, make sure that the extension ID is the same as as the old extesion

### ISSUE THREAD

**Issue by ExE-Boss** [Infocatcher/Private_Tab_WE#2]
https://github.com/Infocatcher/Private_Tab_WE/blob/68dce4ddbcb0fb09aa02f31c2c146b90613ee397/manifest.json#L10 This should instead be `privateTab@infocatcher` as per [MDN documentation (step 4)](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Porting_a_legacy_Firefox_add-on#Quick_start).

This way, everyone who used the legacy extension will get updated to the WebExtension.

**Comment by Infocatcher** [comment]
Yes, but currently WE version isn't functional due to API limitations, so I prefer to have both installed (it's still possible to apply some hacks to use legacy extensions).
And later I'll change ID like here: https://github.com/Infocatcher/Right_Links_WE/commit/25b2c3939b67fcb8e1c7af14ff80d5f10b6aa1fd
